packages/property-grid/src/PropertyGridManager.ts

```diff
@@ -71,3 +80,32 @@ export class PropertyGridManager {
     return category;
   }
 }
+
+/** Extension object for loading on runtime */
+class PropertyGridExtension extends Extension {
```

Comment:
Should the extension work be extracted into its own file?

Reply:
Moved.